sentry-actix/src/lib.rs

```diff
@@ -622,9 +625,9 @@ mod tests {
         }
     }
 
-    /// Ensures client errors (4xx) are not captured.
+    /// Ensures client errors (4xx) returned by service are not captured.
```

Comment:
What is a "service" in this context? Is it a middleware?

[edit] actually if I understand correctly, the service is the endpoint itself, not the middleware. Unless this is common terminology in actix-web, I would clarify the wording in the comment